documentation/webapi.md

```diff
@@ -228,10 +228,66 @@ curl -X "POST" "https://webapi.vvo-online.de/dm/trip" \
 ```
 
 
+# Route Changes
+
+Get information about route changes because of construction work or such.
+
+## Request
+
+POST https://webapi.vvo-online.de/rc
+
+### JSON body
+
+| Name        | Type        | Description    | Required |
+| ----------- | ----------- | -------------- | -------- |
+| shortterm   | Bool        | unknown. I diffed the output with and without -> no diff | no        |
+
+I also tried to pass other keys like mot, name, id, change, ... (each camel, pascal and lower case)
+but no error and no change in result. So it looks like you need to fetch all route changes for the
+whole VVO and filter yourself.
+
+```bash
+curl -X "POST" "https://webapi.vvo-online.de/rc" \
+     -H 'Content-Type: application/json; charset=utf-8' \
+     -d $'{ "shortterm": true }'
+```
+
+## Response
+
+```json
+{
+    "Changes": [
+        ...
+        {
+            "Description": "<p><DIV ></DIV>\n<H2>Beschreibung</H2>\n<P><STRONG>Buslinie 79:</STRONG><BR>Umleitung <FONT color=#ff0000>nur in Richtung&nbsp;Overbeckstraße</FONT> zwischen den Haltestellen&nbsp;Rethelstraße und&nbsp;Mengsstraße über den Fahrweg <U>Rethelstraße - Werftstraße</U>.</P>\n<H2>Haltestellenanpassungen</H2>\n<UL>\n<LI>Die Haltestellen <STRONG>Kaditzer Straße</STRONG> und <STRONG>Thäterstraße</STRONG>&nbsp;werden in die <U>Rethelstraße</U> verlegt.</LI></UL></p>",
+            "Id": "511595",
+            "LineIds": [
+                "428296"
+            ],
+            "PublishDate": "/Date(1512400560000+0100)/",
+            "Title": "Dresden - Mengsstraße, Vollsperrung wegen Asphaltarbeiten",
+            "Type": "Scheduled",
+            "ValidityPeriods": [
+                {
+                    "Begin": "/Date(1512529200000+0100)/",
+                    "End": "/Date(1512788400000+0100)/"
+                }
+            ]
+        },
+        ...
+    ],
+    "Status": {
+        "Code": "Ok"
+    }
+}
+```
+
+(sometimes they come back: `<font color="#abc" />`)
```

Comment:
what do you mean by this?

Reply:
That's just my trying to be funny - sorry for that ;)
"sometimes they come back" is a reference to a movie based on a story [1] of Stephen King.
And I wanted to say: Some bad stuff just never goes away/always comes back. Like the evil `<font/>` [2] tag. In the example response above, it is: `<FONT color=#ff0000>`

Since this is not useful for the wiki I'm totally fine with removing this line.

[1]: https://en.wikipedia.org/wiki/Sometimes_They_Come_Back_(film)
[2]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font

Reply:
Ah 🙈😄 Might as well stay in for now 😄